engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/AvroSchemaDeterminer.scala

```diff
@@ -0,0 +1,13 @@
+package pl.touk.nussknacker.engine.avro
+
+import cats.data.Validated
+import org.apache.avro.Schema
+import pl.touk.nussknacker.engine.avro.schemaregistry.SchemaRegistryError
+
+trait AvroSchemaDeterminer extends Serializable {
+
+  def determineSchemaUsedInTyping: Validated[SchemaRegistryError, Schema]
```

Comment:
I think on this leve we should have another type of error - SchemaRegistry is specific implementation.